services/dask-sidecar/tests/unit/test_settings.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+import subprocess
+
+import pytest
+from simcore_service_dask_sidecar.settings import Settings
+
+
+@pytest.fixture
+def mock_service_envs(mock_env_devel_environment, monkeypatch):
+
+    # Variables directly define inside Dockerfile
+    monkeypatch.setenv("SC_BOOT_MODE", "debug-ptvsd")
+
+    # Variables  passed upon start via services/docker-compose.yml file under dask-sidecar/scheduler
+    monkeypatch.setenv("DASK_SCHEDULER_ADDRESS", "tcp://dask-scheduler:8786")
+
+    monkeypatch.setenv("SWARM_STACK_NAME", "simcore")
+    monkeypatch.setenv("SIDECAR_LOGLEVEL", "WARNING")
+    monkeypatch.setenv("SIDECAR_HOST_HOSTNAME_PATH", "/home/scu/hostname")
+    monkeypatch.setenv("START_AS_MODE_CPU", "0")
+
+
+def test_cli_start_dask_worker(mock_service_envs):
+
+    settings = Settings.create_from_envs()
+    assert settings.as_worker()
+
+    subprocess.run(["dask-worker", "--version"], check=True)
```

Comment:
what are you testing here? not sure I get it. isn't there some check missing?

Reply:
since ``check=True``, when the exit status code is not zero, it will raise an exception ``subprocess.CalledProcessError``

perhaps it is more readable writing
```python
assert subprocess.run(["dask-worker", "--version"]).returncode == os.EX_OK
```

Reply:
in any case, this test is meaningless. I will remove it.